README.md

```diff
@@ -123,7 +123,10 @@ URL specified in the definition file.
 You can point ruby-build to another mirror by specifying the
 `RUBY_BUILD_MIRROR_URL` environment variable--useful if you'd like to run your
 own local mirror, for example. Package mirror URLs are constructed by joining
-this variable with the SHA2 checksum of the package file.
+this variable with the SHA2 checksum of the package file. For example, the default
+URL for `ruby-2.3.0.tar.bz2` is http://cache.ruby-lang.org/pub/ruby/ruby-2.3.0.tar.bz2.
+That package would be mirrored at `$RUBY_BUILD_MIRROR_URL/ec7579eaba2e4c402a089dbc86c98e5f1f62507880fd800b9b34ca30166bfa5e`,
```

Comment:
I'm sorry, but I don't think this example clarifies anything over what was already being said.

“Package mirror URLs are constructed by joining [RUBY_BUILD_MIRROR_URL] with the SHA2 checksum of the package file” → `<RUBY_BUILD_MIRROR_URL>/<SHA2>`. Was this unclear?

Reply:
Yes. I wasn't expecting moving from the usual "file with an extension" to download to "arbitrary path with no extension". It took me some time and stepping through the script to realize it wasn't `$RUBY_BUILD_MIRROR/$SHASUM/$PACKAGE`.